index.d.ts

```diff
@@ -317,12 +319,12 @@ export class CommanderError extends Error {
     /**
      * Set the custom handler for processing CLI option arguments into option values.
      */
-    argParser<T>(fn: (value: string, previous: T) => T): Option<Usage, PresetT, DefaultT, T, Mandatory>;
+    argParser<T>(fn: (value: string, previous: T) => T): Option<Usage, PresetT, DefaultT, T, Mandatory, ChoicesT>;
```

Comment:
Last one wins out of `choices` and `argParser`, so should pass in undefined for `ChoicesT` here?

(Noticed because of your comment about passing in undefined for `argParser` in `choices()`.)